internal/pipe/changelog/changelog.go

```diff
@@ -121,6 +121,28 @@ func buildChangelog(ctx *context.Context) ([]string, error) {
 	entries := strings.Split(log, "\n")
 	entries = entries[0 : len(entries)-1]
 	entries, err = filterEntries(ctx, entries)
+
+	for index, entry := range entries {
+		splitEntry := strings.Split(entry, " ")
+		lastElement := splitEntry[len(splitEntry)-1]
+		if strings.HasPrefix(lastElement, "(#") &&
+			strings.HasSuffix(lastElement, ")") {
+			commitHash := strings.Split(entry, " ")[0]
+			authorName, err := gitAuthorName(commitHash)
+			if err != nil {
+				return entries, err
+			}
+			splittedAuthorName := strings.Split(authorName, "\n")
+			splittedAuthorName = splittedAuthorName[0 : len(splittedAuthorName)-1]
+			authorName = splittedAuthorName[0]
+			if err != nil {
+				return entries, err
+			}
+			entries[index] = fmt.Sprintf("%s - Thanks to %s !", entry, authorName)
```

Comment:
It would be great if we provide an ability to customize author format. What about making it configurable from a template? (in `config.go`) i.e., `%s - by Foo`, `%s - Kudos to Bar`, etc.

Reply:
we had talked on this issue, I think we should resolve it.